CITATION.cff

```diff
@@ -1,20 +1,41 @@
 cff-version: 1.2.0
-message: "If you use this software, please cite it as below."
+message: |
+  Please cite this software using these metadata. "*" indicates co-equal authorship.
+
+title: "QGreenland"
+doi: "10.5281/zenodo.4558266"
+abstract: "This software generates the QGreenland data package for QGIS."
+contact:
+  - name: "The QGreenland Team"
+    email: "qgreenland.info@gmail.com"
+  - name: "The QGreenland community" 
+    website: "https://github.com/nsidc/qgreenland/discussions"
+type: "software"
+license: ["MIT"]
+keywords:
+  - "GIS"
+  - "QGIS"
+  - "Python"
+  - "education"
```

Comment:
"Greenland" might be another good keyword :)

Reply:
```suggestion
  - "education"
  - "Greenland"
```

Reply:
Why do you say that? ;)